Makefile

```diff
@@ -360,10 +361,11 @@ kind-operator-down: $(KIND)
 export SKAFFOLD_BUILD_CONCURRENCY = 0
 gardener%up gardener%dev gardener%debug gardenlet%up gardenlet%dev gardenlet%debug operator-up operator-dev operator-debug: export SKAFFOLD_DEFAULT_REPO = localhost:5001
 gardener%up gardener%dev gardener%debug gardenlet%up gardenlet%dev gardenlet%debug operator-up operator-dev operator-debug: export SKAFFOLD_PUSH = true
+gardener%up gardener%dev gardener%debug gardenlet%up gardenlet%dev gardenlet%debug operator-up operator-dev operator-debug: export SOURCE_DATE_EPOCH = $(shell date -d $(BUILD_DATE) +%s)
 # use static label for skaffold to prevent rolling all gardener components on every `skaffold` invocation
 gardener%up gardener%dev gardener%debug gardener%down gardenlet%up gardenlet%dev gardenlet%debug gardenlet%down: export SKAFFOLD_LABEL = skaffold.dev/run-id=gardener-local
 # set ldflags for skaffold
-gardener%up gardener%dev gardener%debug gardenlet%up gardenlet%dev gardenlet%debug operator-up operator-dev operator-debug: export LD_FLAGS = $(shell $(REPO_ROOT)/hack/get-build-ld-flags.sh)
+gardener%up gardener%dev gardener%debug gardenlet%up gardenlet%dev gardenlet%debug operator-up operator-dev operator-debug: export LD_FLAGS = $(shell $(REPO_ROOT)/hack/get-build-ld-flags.sh k8s.io/component-base $(REPO_ROOT)/VERSION $(BUILD_DATE))
```

Comment:
Shouldn't `$(BUILD_DATE)` be on the 4th position of all parameters?

Reply:
Why do we need to pass it at all when its already defaulted to the same value in the script?

Reply:
Hm, I don't understand 😄 

Here, `$(BUILD_DATE)` is passed the **3rd** parameter.

In the `hack/get-build-ld-flags.sh` script however, BUILD_DATE is defined as the **4th** parameter and PROGRAM_NAME as the **3rd**:

```
PROGRAM_NAME="${3:-Gardener}"
BUILD_DATE="${4:-$(date '+%Y-%m-%dT%H:%M:%S%z' | sed 's/\([0-9][0-9]\)$/:\1/g')}"
```

It seems like `PROGRAM_NAME` gets assigned `$(BUILD_DATE)`? Am I missing something?

Reply:
Yeah you are correct thanks. Updated